docs/dunst.5.pod

```diff
@@ -106,6 +106,15 @@ the notification on the left border of the screen while a horizontal offset of
 
 =back
 
+=item B<scale> (default: 0)
```

Comment:
Could you add `(X11 only)` here as well?

Reply:
Done.